packages/extract-react-types/__snapshots__/converters-typescript.test.js.snap

```diff
@@ -1709,171 +1745,6 @@ Object {
 }
 `;
 
-exports[`TypeScript: typescript React.ComponentType 1`] = `
-Object {
-  "component": Object {
-    "kind": "generic",
-    "name": Object {
-      "kind": "id",
-      "name": "Component",
-      "type": null,
-    },
-    "value": Object {
-      "kind": "object",
-      "members": Array [
-        Object {
-          "key": Object {
-            "kind": "id",
-            "name": "children",
-          },
-          "kind": "property",
-          "optional": false,
-          "value": Object {
-            "kind": "generic",
-            "value": Object {
-              "kind": "id",
-              "name": "React.ComponentType",
-            },
-          },
-        },
-      ],
-      "referenceIdName": "Props",
-    },
-  },
-  "kind": "program",
-}
-`;
-
-exports[`TypeScript: typescript React.FC 1`] = `
-Object {
-  "component": Object {
-    "kind": "generic",
-    "name": Object {
-      "kind": "id",
-      "name": "Component",
-      "type": null,
-    },
-    "value": Object {
-      "kind": "object",
-      "members": Array [
-        Object {
-          "key": Object {
-            "kind": "id",
-            "name": "children",
-          },
-          "kind": "property",
-          "optional": false,
-          "value": Object {
-            "kind": "string",
-          },
-        },
-      ],
-      "referenceIdName": "Props",
-    },
-  },
-  "kind": "program",
-}
-`;
-
-exports[`TypeScript: typescript React.FC with empty type argument 1`] = `
-Object {
-  "component": Object {
-    "kind": "any",
-    "name": Object {
-      "kind": "id",
-      "name": "Component",
-      "type": null,
-    },
-  },
-  "kind": "program",
-}
-`;
-
-exports[`TypeScript: typescript React.FC with inline type argument 1`] = `
-Object {
-  "component": Object {
-    "kind": "object",
-    "members": Array [
-      Object {
-        "key": Object {
-          "kind": "id",
-          "name": "children",
-        },
-        "kind": "property",
-        "optional": false,
-        "value": Object {
-          "kind": "string",
-        },
-      },
-    ],
-    "name": Object {
-      "kind": "id",
-      "name": "Component",
-      "type": null,
-    },
-  },
-  "kind": "program",
-}
-`;
-
-exports[`TypeScript: typescript indexed imported type 1`] = `
-Object {
-  "component": Object {
-    "kind": "object",
-    "members": Array [
-      Object {
-        "key": Object {
-          "kind": "id",
-          "name": "foo",
-        },
-        "kind": "property",
-        "optional": false,
-        "value": Object {
-          "kind": "generic",
-          "value": Object {
-            "kind": "id",
-            "name": "ImportedType['props']",
-          },
-        },
-      },
-    ],
-    "name": Object {
-      "kind": "id",
-      "name": "Component",
-      "type": null,
-    },
-  },
-  "kind": "program",
-}
-`;
-
-exports[`TypeScript: typescript indexed type 1`] = `
```

Comment:
These have been removed because they are duplicates. Caused by a rename (removing the second typscript)